validation/src/main/scala/hmda/validation/rules/lar/validity/V300.scala

```diff
@@ -0,0 +1,66 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.census.CBSATractLookup
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.PredicateRegEx._
+
+object V300 extends EditCheck[LoanApplicationRegister] {
+
+  val cbsaTracts = CBSATractLookup.values
+
+  val validCombination = cbsaTracts.map { cbsa =>
+    (cbsa.geoIdMsa, cbsa.state, cbsa.county, cbsa.tractDecimal)
+  }
+
+  val validMdCombination = cbsaTracts.map { cbsa =>
+    (cbsa.metDivFp, cbsa.state, cbsa.county, cbsa.tractDecimal)
+  }
+
+  val validStateCountyCombination = cbsaTracts.map { cbsa =>
+    (cbsa.state, cbsa.county, cbsa.tractDecimal)
+  }
```

Comment:
Hey Laura, if there are advantages to be gotten I will happily switch over! However, would you mind explaining a bit more why it would be faster or pointing me to some documentation? Thanks!